diff --git a/node_modules/@strapi/plugin-users-permissions/server/content-types/permission/index.js b/node_modules/@strapi/plugin-users-permissions/server/content-types/permission/index.js
index 70648c8..90d1143 100644
--- a/node_modules/@strapi/plugin-users-permissions/server/content-types/permission/index.js
+++ b/node_modules/@strapi/plugin-users-permissions/server/content-types/permission/index.js
@@ -9,12 +9,13 @@ module.exports = {
     pluralName: 'permissions',
     displayName: 'Permission',
   },
+  options: {},
   pluginOptions: {
     'content-manager': {
-      visible: false,
+      visible: true,
     },
     'content-type-builder': {
-      visible: false,
+      visible: true,
     },
   },
   attributes: {
diff --git a/node_modules/@strapi/plugin-users-permissions/server/content-types/role/index.js b/node_modules/@strapi/plugin-users-permissions/server/content-types/role/index.js
index ceb45c6..4c70fa8 100644
--- a/node_modules/@strapi/plugin-users-permissions/server/content-types/role/index.js
+++ b/node_modules/@strapi/plugin-users-permissions/server/content-types/role/index.js
@@ -9,12 +9,13 @@ module.exports = {
     pluralName: 'roles',
     displayName: 'Role',
   },
+  options: {},
   pluginOptions: {
     'content-manager': {
-      visible: false,
+      visible: true,
     },
     'content-type-builder': {
-      visible: false,
+      visible: true,
     },
   },
   attributes: {
